diff --git a/src/libsnore/snore_static_plugins.h.in b/src/libsnore/snore_static_plugins.h.in
index af801e9..baccd41 100644
--- a/src/libsnore/snore_static_plugins.h.in
+++ b/src/libsnore/snore_static_plugins.h.in
@@ -31,15 +31,15 @@ namespace SnorePlugin {}
 using namespace SnorePlugin;
 ${SNORE_PLUGIN_LOADING}
 
-namespace {
-    static void loadSnoreResources()
-    {
-        // prevent multiple symbols
-         static const auto load = []() {
-             ${SNORE_RESOURCE_LOADING}
-         };
-         load();
-    }
+inline void loadSnoreResources()
+{
+     // prevent multiple symbols
+     static const auto load = []() {
+         ${SNORE_RESOURCE_LOADING}
+         return 0;
+     }();
+     // prevent annoying warnings
+     (void)load;
 }
 Q_COREAPP_STARTUP_FUNCTION(loadSnoreResources)
 #endif
